src/Feliz.Plotly/Feliz.Plotly.fsproj

```diff
@@ -5,6 +5,18 @@
     <PackageId>Feliz.Plotly.V2</PackageId>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Version>2.28.0</Version>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Authors>Kurt Mueller</Authors>
+    <PackageTags>fsharp fable react visualization graph chart</PackageTags>
+    <PackageDescription>Fable bindings written in the Feliz-style for plotly.js</PackageDescription>
```

Comment:
This data was all added by paket.template to the previous package, now it's supplied for dotnet pack as well.